### ISSUE TITLE
NgEntityService upsert for get without id

### ISSUE THREAD

**Issue by ntziolis** [datorama/akita#306]
## I'm submitting a...
<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Performance issue
[x] Feature request
[ ] Documentation issue or request
[ ] Support request
[ ] Other... Please describe:
</code></pre>

## Current behavior
When using the calling `get()` on an `NgEntityService` the result either:
- replaces the data currently in the store 
- or new data can be appended (this will not update existing records).

## Expected behavior
It should also be possible choose an option that does:
- append new records
- and update existing ones
- without removing records that are NOT part of the returned data

In other words the equivalent of calling upsert for each records contained in the response data.

## Minimal reproduction of the problem with instructions
The behaviour for append / replace can be seen here:
https://github.com/datorama/akita/blob/14510aa6294bae7a91547bdb02813817d8cb5ac0/angular/playground/projects/ng-entity-service/src/lib/ng-entity.service.ts#L104

## What is the motivation / use case for changing the behavior?
We always wanne have the latest returned version of each record in the store.

**Comment by NetanelBasal** [comment]
Did you mean `upsertMany`?

**Comment by NetanelBasal** [comment]
Added in the latest version. https://github.com/datorama/akita/blob/master/angular/playground/projects/ng-entity-service/src/lib/ng-entity.service.ts#L109

**Comment by ntziolis** [comment]
Wow you are damn fast. Just for my sanity this wasn't there 20min ago, correct?

**Comment by NetanelBasal** [comment]
It wasn't.
